### PR TITLE
Via Oberta Authorization

### PR DESCRIPTION
**Pull request by LauraJaimeC**
DOCS:
https://github.com/ConsorciAOC/PADRO
https://github.com/ConsorciAOC/PCI

### CODE REVIEW

File: decidim-via_oberta_authorization/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Decidim::ViaObertaAuthorization
+
+A decidim package to provice user authorizations agains Via Oberta WS
```
Comment (tramuntanal) [inline decidim-via_oberta_authorization/README.md:3]:
```suggestion
A Decidim module to provide user authorizations against Via Oberta WS.
```

File: decidim-via_oberta_authorization/app/services/via_oberta_authorization_config.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    def purpose
+      if Rails.env == :production
+        "GESTTRIB"
+      else
+        "PROVES"
+      end
+    end
+
+    # TODO: add to /system
```
Comment (tramuntanal) [inline decidim-via_oberta_authorization/app/services/via_oberta_authorization_config.rb:24]: We must open a BC task about this...

**Review by tramuntanal — COMMENTED** [review]
Great job here!
Just a couple of things

File: decidim-via_oberta_authorization/app/services/via_oberta_authorization_config.rb (hunk 2 above)
Comment (LauraJaimeC) [reply decidim-via_oberta_authorization/app/services/via_oberta_authorization_config.rb:24]: Done :)

**Review by tramuntanal — APPROVED** [review]
LGTM @LauraJaimeC !

**Pull request merged**